org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/engine/connected/ConnectedEngineFacadeManager.java

```diff
@@ -316,6 +317,23 @@ private static void tryRemoveSecureProperties(String serverNodeName) {
     return Optional.ofNullable(facadesByConnectionId.get(Objects.requireNonNull(id)));
   }
 
+  public List<IConnectedEngineFacade> findByUrl(String serverUrl) {
+    return facadesByConnectionId.values().stream()
+        .filter(facade -> equalsIgnoringTrailingSlash(facade.getHost(), serverUrl))
+        .collect(Collectors.toList());
+  }
+
+  private static boolean equalsIgnoringTrailingSlash(String aString, String anotherString) {
+    return withTrailingSlash(aString).equals(withTrailingSlash(anotherString));
+  }
+
+  private static String withTrailingSlash(String str) {
```

Comment:
In other places of the code, we are using ` StringUtils.removeEnd(url, "/")` to normalize URLs.